app/welcome/payment-info/controller.js

```diff
@@ -0,0 +1,7 @@
+import Ember from "ember";
+
+export default Ember.Controller.extend({
+  months: ['01', '02', '03', '04', '05', '06', '07', '08', '09', '10', '11', '12'],
+  years: ['2015', '2016', '2017', '2018', '2019', '2020'],
+  plans: ['development', 'pilot', 'platform', 'production']
```

Comment:
When details like this are stored on a controller, I usually end up creating components. For example, `months` is used in a select box. So I suggest a `select-months` component that internalizes the list of options. This is much more re-usable and semantic.